images/linux/scripts/installers/git.sh

```diff
@@ -8,14 +8,13 @@
 source $HELPER_SCRIPTS/install.sh
 
 ## Install git
-add-apt-repository ppa:git-core/ppa -y
 apt-get update
 apt-get install git -y
 git --version
 
 # Install git-lfs
-curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
-apt-get install -y --no-install-recommends git-lfs
+apt-get update -y
```

Comment:
I think we don't need to run apt-get update one more time. We have already run it on line 11